### DISCUSSION TITLE
Does the random charge cycling have an effect on the calibrated capacity of the battery?

### DISCUSSION THREAD

**Discussion by kme77** [nathanmarlor/foxess_em#109] [Q&A]
I have only had my system installed for a month but already noticed that my battery capacity has started to misreport up to the point where the other day it suddenly dropped 20% over a 90min with no associated load. This frustratingly left me without enough charge to cover my evening use. I have asked on the FB group and it was suggested that my battery needs recalibrating with a couple of full charge/discharge cycles.

Is it possible that by using the integration to 'under-charge' overnight that it is effecting the battery calibration and if so is there a way to mitigate this effect either systematically or with planned cycling? 

Also, any other suggestions as to why my battery is misreporting it's charge would be welcomed! I LOVE this integration and obviously hope that the issue lies elsewhere! 👍

**Comment by spmaid** [comment]
Hello!  I have had serious problems with this, losing SoC for no reason, jumping from 75% to 100% in a matter of minutes, things like that. I have been on to FoxESS support about it since early summer, after a a few updates (one went really bad!) I think I finally have a stable and working firmware update on the batteries now.  Note it is a battery firmware update not the inverter.  I was updated to Battery Version 01.00.01.09, if you look on the inverter in about I think it shows it.   I'd suggest dropping them an email, it can take quite a while to get a response.  I have been dealing directly with david.gu@fox-ess.com and you could try service@fox-ess.com too

**Reply by kme77** [reply]
That sounds promising, thank you! I'll check my FW and email them if it's out of date. 👍 What is involved in the update process?
Also, is you're system 'cured' of this nasty little habit now?

**Reply by spmaid** [reply]
David at FoxESS just asked me if he could update, I said yes, then it happens in the background.  Since I have been on this 1.0.1.9 I'd consider the performance to be exactly as I'd expect.  Before, I'd randomly lose SoC overnight, I'd charge for say an hour but put 60% SoC in (which was not possible as I have 7 batteries) so I hours charge should equate to approx. 30%. I was only seeing about 40% of the total storage being usable.  It'd jump from 75% to 100% SoC in around 5 minutes. When it got to 19% SoC it'd start a 1.5kW charge from grid.  All of that has now stopped.

I have had a varied week of having some solar during the day, to none and therefore using all of the battery storage and then fully charging overnight and so far, I am seeing expected levels of charge/discharge each day.  Finally! :-)

**Reply by kme77** [reply]
That's great, thanks I'll definitely look into that!

**Reply by kme77** [reply]
![IMG_20221204_105447](https://user-images.githubusercontent.com/38857063/205487220-429f4f09-0b42-42df-aa2b-05c906fb9f68.jpg)

Just looking at my info, does this correspond to firmware v1.0.1.5 on both of the batteries? (I assume the first 1 corresponds to the battery position?)

**Reply by spmaid** [reply]
Apologies yes, the first number is battery position. 🤦 I was writing it from memory. So the firmware version update I got was 1.09 
![Uploading FA2536DC-57D4-4DBA-AA9D-890F20CA5B73.jpeg…]()

**Reply by spmaid** [reply]
Tried posting a photo to show it, I also have M showing 1.11 it seems

**Reply by kme77** [reply]
Looks like the photo failed to upload for some reason 🤷‍♂️

M?

**Reply by spmaid** [reply]
![CF30906E-27E7-475B-BB87-0FE5EA0AAA45](https://user-images.githubusercontent.com/99452615/205489137-4acba4a6-6948-411d-a4a8-2aba9859cd60.jpeg)

**Reply by kme77** [reply]
Interesting, has your inverters firmware been updated too?

**Reply by spmaid** [reply]
I had a firmware upgrade not long after I had the Inverter for another small issue, not sure if this helps/indicates but App and Inverter currently showing:

Master 1.48
Slave 1.02
Manager Version 1.47

I’m not aware of that being updated recently as pretty sure it’s been at those values for a while.
